src/application/redux/containers/choose-fee.container.ts

```diff
@@ -13,7 +13,7 @@ const mapStateToProps = (state: RootReducerState): ChooseFeeProps => ({
   taxiAssets: state.taxi.taxiAssets,
   lbtcAssetHash: lbtcAssetByNetwork(state.app.network),
   sendAddress: state.transaction.sendAddress,
-  changeAddress: state.transaction.changeAddress,
+  changeAddress: state.transaction.changeAddresses[0],
```

Comment:
why we assume is the first element?

Reply:
we always send 1 asset in the popup so it means that we have either 0 or 1 change address (note that `changeAddress` prop can be undefined).